### PR TITLE
update/delete database

### PR DESCRIPTION
**Pull request by rushali30**

### CODE REVIEW

**Comment by rushali30** [comment]
Update/delete functions

File: src/csps/src/coachsafeplaysafe.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 import os
 
 
-no_sensor_api = False
+mnnno_sensor_api = False
```
Comment (srikumar947) [inline src/csps/src/coachsafeplaysafe.py:15]: mnno_sensor? Have you modified screen two code?

**Review by srikumar947 — CHANGES_REQUESTED** [review]
Why are you committing pyc (compiled python) and db files?

**Comment by rhk12** [comment]
@srikumar947 you might need to make the changes.I am not sure if Rushali will do it.

**Pull request closed without merging**